docs/core/testing/index.md

```diff
@@ -38,9 +48,8 @@ You can read about those combinations in these walkthroughs:
 * Create unit tests using [*NUnit* and *Visual Basic* with the .NET Core CLI](unit-testing-visual-basic-with-nunit.md).
 * Create unit tests using [*MSTest* and *Visual Basic* with the .NET Core CLI](unit-testing-visual-basic-with-mstest.md).
 
-You can choose different languages for your class libraries and your unit test libraries. You can learn how by mixing and matching the walkthroughs referenced above.
+Further readings related to testing:
```

Comment:
I'd rephrase this as "You can learn more in the following articles."

Reply:
I agree.